scripts/Jenkinsfile

```diff
@@ -80,6 +80,10 @@ node {
                   def testdata = readFile(file: filename)
                   println("\n\nTFVARS TEST DATA")
                   println(testdata)
+                  if ((env.PRODUCT_NAME == 'k3s' && TESTCASE == 'mixedoscluster') || (env.PRODUCT_NAME == 'rke2' && testdata.contains('cis'))) {
+                    error('Unsupported test execution found, Aborting the build!')
```

Comment:
We can explicitly specify which test combination is not allowed in the error message for better comprehension of why this errored.

Reply:
Updated, please review